{
  "comments": [
    {
      "unresolved": true,
      "key": {
        "uuid": "a06bdaf5_6bd7cb94",
        "filename": "/COMMIT_MSG",
        "patchSetId": 1
      },
      "lineNbr": 10,
      "author": {
        "id": 1644679
      },
      "writtenOn": "2024-01-13T01:03:46Z",
      "side": 1,
      "message": "I guess there is no point in rerunning the failed tests in b/247689681 and b/319340965 with this CL because AIDL is now disabled again?",
      "range": {
        "startLine": 10,
        "startChar": 0,
        "endLine": 10,
        "endChar": 7
      },
      "revId": "45906c7d9a4ba85f034c65426f9493e590578d2f",
      "serverId": "85c56323-6fa9-3386-8a01-6480fb634889"
    }
  ]
}